Travel_fellowships.md

```diff
@@ -1,16 +1,18 @@
 # Open Bioinformatics Foundation Event Fellowships
 
 The Open Bioinformatics Foundation ([OBF](https://www.open-bio.org)) promotes open source bioinformatics software development and open science in the biological research community. 
-The OBF Event Fellowship program is aimed at increasing diverse participation at events promoting open science practices such as open source software and resource development in the bioinformatics and biological research community.
+The OBF Event Fellowship program is aimed at increasing diverse participation and representation at events promoting open science practices such as open source software and resource development in the bioinformatics and biological research community.
 
 These fellowships are available to support both in-person and remote (virtual) participation at events such as conferences, workshops, training courses or collaborative development sprints.
 
 ## Application deadlines
 
-There are two rounds of applications with the following **application deadlines**:
+There are several rounds of applications with the following **application deadlines**:
```

Comment:
Maybe note that these dates may change in future rounds?

Reply:
Would you like to rephrase this?